src/Samhammer.AzureBlobStorage/Options/AzureBlobStorageOptions.cs

```diff
@@ -5,5 +5,7 @@ public class AzureBlobStorageOptions
         public string ConnectionString { get; set; }
 
         public string ContainerName { get; set; }
+
+        public int FileUrlExpiresByDays { get; set; } = 1;
```

Comment:
@khoanguyen102 this setting looks unusedto me? do i oversee something or is the usage missing?

Reply:
I add the options for settings the expiration time of URL. It is configured in appsettings in Asimov @agentschmitt

Reply:
I have updated it @agentschmitt, thanks :) I will update version and change in Asimov

Reply:
@khoanguyen102 kittel has mentioned that he would like to have some shorter url lifetime e.g. 12 hours so the links would probably be valid for the time of one workday only. Can you change the Option to support Hours instead of Days? Or alternative support set of a TimeSpan from configuration that allows both then ;)
appsetting would look like this for one day "FileUrlExpires": "1.00:00:00",

Reply:
Thanks Roli, it sound great, I have updated it

Reply:
@khoanguyen102 i updated the readme as the sample timespan throw some parse error :)

Reply:
ok i failed somehow on testing, the timespan was valid so i changed it back.